src/ts/toasts.ts

```diff
@@ -0,0 +1,44 @@
+import {idiom as lang} from "./idiom";
+import {$} from "./entcore";
+
+/**
+ * @description a pop-up on the right screen to display infos or more
+ * @param text :message which will be displayed
+ */
+export const toasts =  {
+    message: function(status, text){
+        let HasDivToasts=$(".toasts")[0];
+        if(!HasDivToasts){
+            $("body").append('<div class="toasts"></div>');
+        }
+        text = lang.translate(text);
+        let toastsContainer = $('.toasts')[0];
+        let toastTemplate = `
+        <div class="show vertical-spacing horizontal-spacing ${status} toast-content">
+            <div class="content">
+                ${lang.translate(text)}
+            </div>
+            <div class="timer animate"></div>
+        </div>
+    `;
+    let  toastTemplateJquery = $(toastTemplate)
```

Comment:
y'a un espace en trop après le let :(

```suggestion
    let toastTemplateJquery = $(toastTemplate);
```